.eslintrc.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
     "import", // require("eslint-plugin-import")
     "json", // require("eslint-plugin-json")
     "promise", // require("eslint-plugin-promise")
-    "react" // require("eslint-plugin-react")
+    "react", // require("eslint-plugin-react")
+    "react-hooks"
```

Comment:
Probably good to add the fake require comment as the previous lines have to make dumb simple grepping work.

Reply:
Sure.  Out of curiousity, in what circumstances does one grep for this?  Just
when manually checking to see if this module is still used?